index.js

```diff
@@ -331,3 +600,62 @@ function destroy() {;
   return Promise.all(promises);
 }
 
+// Lookup the conversations for the calls to get their title
+async function lookupConversations(client, calls) {
+  let promises = [];
+  calls.forEach(call => {
+    promises.push(client.getConversationById(call.convId));
+  });
+  return Promise.all(promises)
+  .then(res => {
+    return res.map((res, idx) => {
+      // Used to be res.title (copied from JSBin examples) 
+      // but there isn't a title attribute in the payload.
+      // If there is more than one conf with same name we 
+      // include the placeholder to show what users are
+      // in the conference so the user knows which is 
+      // which. 
+      return {title: res.topic, callId: calls[idx].callId, topicPlaceholder: res.topicPlaceholder, participants: res.participants};
+    });
+  });
+}
+
+/**
+ * Returns all necessary conference info needed for joining and leaving a conference.
+ */
+async function getConferenceInfo(conv){
+  const circuit = await getCircuit(conv);
+  if (!circuit) {
+    return;
+  }
+
+  let startedCalls = await circuit.getStartedCalls(),
+  calls = await lookupConversations(circuit, startedCalls),
```

Comment:
New line for each please. More readable. And use const whenever possible.

`const calls = await lookupConversations(circuit, startedCalls);
`